### ISSUE TITLE
distance weapons missing defense description

### ISSUE THREAD

**Issue by emil92b** [otland/forgottenserver#3687]
### Before creating an issue, please ensure:
- [x] This is a bug in the software that resides in this repository, and not a
      support matter (use https://otland.net/forums/support.16/ for support)
- [x] This issue is reproducible without changes to the C++ code in this repository

### Expected behaviour
09:22 You see a spear (Atk:25, Def:0).
It weighs 20.00 oz.

### Actual behaviour
09:22 You see a spear (Atk:25).
It weighs 20.00 oz.

**Comment by EPuncker** [comment]
yeah but we will need to set `<attribute key="defense" value="0" />` to every item we want to show it that way, otherwise we would get that defense 0 being show to every item, and also while we are at this, we should also add attack 0.


```
You see a snowball (Atk:0, Def:0).
It weighs 0.80 oz.
```

**Comment by marciNekel** [comment]
Maybe check if there is attack attribute and no def attribute then push description with Def: 0?
